proto/storage/compliance_integration.proto

```diff
@@ -5,12 +5,13 @@ package storage;
 option go_package = "storage";
 option java_package = "io.stackrox.proto.storage";
 
-// Next Tag: 6
+// Next Tag: 7
 message ComplianceIntegration {
   string id = 1; // @gotags: sql:"pk,type(uuid)"
   string version = 2; // @gotags: search:"Compliance Operator Version,hidden,store"
   string cluster_id = 3; // @gotags: search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid),index=category:unique;name:compliance_unique_indicator"
   string compliance_namespace = 4;
   // Collection of errors that occurred while trying to obtain compliance health info.
   repeated string status_errors = 5;
+  bool operator_installed = 6;
```

Comment:
Thinking ahead we could make this a searchable field so anyone calling the `ListComplianceIntegrations` could narrow their responses

Reply:
```
bool operator_installed = 6; // @gotags: search:"Operator Installed,hidden"
```

`store` is not longer required there.  I would also recommend making the tag `Compliance Operator Installed`

If you wanted to reference the tag in code and/or a unit test you would also need to make a corresponding update to https://github.com/stackrox/stackrox/blob/bac6c6ab0c48d6f9bbac1ffbdfac194137c55d94/pkg/search/options.go#L209

Reply:
Added it :+1: